docs/use/myst.md

```diff
@@ -227,6 +229,104 @@ which automatically numbers the labelled objects.
 For example, `` {numref}`my_figure` `` will produce a reference like:
 {numref}`my_figure`.
 
+### Sidebar captions and figures
+
+You can include a figure caption on the sidebar using `:figclass: sidebar-caption` as seen in {numref}`sidebar_caption_figure`.
+
+```{figure} ../images/cool.jpg
+---
+height: 150px
+figclass: sidebar-caption
+name: sidebar_caption_figure
+---
+Here is my figure caption!
+```
+
+Another option is to include figures on the sidebar using `:figclass: sidebar` as seen in {numref}`sidebar_figure`.
+
+```{figure} ../images/cool.jpg
+---
+width: 60%
+figclass: sidebar
+name: sidebar_figure
+---
+Here is my figure caption!
+```
+
+### Figure scaling and alignment
+
+Figures can also be aligned by using the option `:align: right` or `:align: left`. By default, figures are aligned to the center (see {numref}`my_figure`).
+
+````
+```{figure} ../images/cool.jpg
+---
+scale: 50%
+align: left
+---
+Here is my figure caption!
+```
+````
+
+```{figure} ../images/cool.jpg
+---
+scale: 50%
+align: left
+---
+Here is my figure caption!
+```
+
+````
+```{figure} ../images/cool.jpg
+---
+scale: 50%
+align: right
+---
+Here is my figure caption!
+```
+````
+
+```{figure} ../images/cool.jpg
+---
+scale: 50%
+align: right
+
+---
+Here is my figure caption!
+```
+### Figure Parameters
+
+The following options are supported:
+
+* `scale` : _integer percentage_
+
+  Uniformly scale the figure. The default is "100" which indicates no scaling. The symbol "%" is optional.
+
+* `width` : _length or percentage_
+
+  You can set the figure width in the following units: "em", "ex", "px","in" ,"cm", "mm", "pt", "pc", "%".
+
+* `height` : _length_
+
+  You can set the figure height in the following units: "em", "ex", "px", "in", "cm", "mm", "pt", "pc", "".
+
+* `alt` : _text_
+
+  Text to be displayed if the figure cannot display, or if the reader is using assistive technologies. Generally entails a short description of the figure.
+
+* `align` : _"left", "center", or "right"_
+
+  Align the figure left, center, or right. Default alignment is center.
+
+* `name` : _text_
+
+  A unique identifier for your figure that you can use to reference it with `{ref}` or `{numref}`. Cannot contain spaces or special characters.
+
+* `figclass` : _text_
+
+  Value of the figure's class attribute which can be used to add custom CSS or JavaScript. Predefined options include:
+  * _"sidebar"_ : Display figure under sidebar
```

Comment:
note - these should now be "margin" and "margin-caption". Sorry about the confusion there, this was just changed in sphinx-book-theme

Reply:
They'll still rendered under sidebar, correct? I'm wondering if it makes sense to change "Display figure under __sidebar__" to "Display figure under __margin__".

Reply:
yep, agreed - in general the white space to the right of the text we are now referring to as "margin" instead of sidebar (to avoid confusion with other UI elements)

Reply:
sounds good!